plugins/follow_me/follow_me.h

```diff
@@ -0,0 +1,194 @@
+        ///////////////////////////////////////////////////////////
+
+        // FollowMe Configuration Accessors.
+        /**
+         * @brief Gets minimum follow target height, in meters.
+         * @return Minimum follow target height.
+         * @sa set_min_height_m(float)
+         */
+        float min_height_m() const;
+
+        /**
+         * @brief Gets distance to follow target from, in meters.
+         * @return Current distance to follow target from, in meters.
+         * @sa set_follow_target_dist_m(float)
+         */
+        float follow_target_dist_m() const;
+
+        /**
+         * @brief Gets Side to follow target from.
+         * @return %FollowDirection representing Side to follow target from.
+         * @sa set_follow_dir(FollowDirection), to_str(FollowDirection)
+         */
+        FollowDirection follow_dir() const;
+
+        /**
+         * @brief String form of %FollowDirection
+         * @param dir enum %FollowDirection
+         * @return std::string representation of enum %FollowDirection
+         * @sa follow_dir(), set_follow_dir(FollowDirection)
+         */
+        static std::string to_str(FollowDirection dir);
+
+        /**
+         * @brief Gets current dynamic filtering algorithm responsiveness value.
+         * @return Current responsiveness value
+         * @sa set_responsiveness(float)
+         */
+        float responsiveness() const;
+
+        // Methods that configure FollowMe behavior.
+        /**
+         * @brief Sets minimum follow target height, in meters.
+         * @param mht: min height in meters relative to home for following a target.
+         * @return true (success) if given height >= 8.0 meters, false (failure) otherwise.
+         * @sa min_height_m()
+         */
+        bool set_min_height_m(float mht);
+
+        /**
+         * @brief Sets distance in meters to follow target from.
+         * @param ft_dst distance in meters to follow the target at.
+         * @return true (success) if given distance >= 1.0 meters, false (failure) otherwise.
+         * @sa follow_target_dist_m()
+         */
+        bool set_follow_target_dist_m(float ft_dst);
+
+        /**
+         * @brief Sets the side to follow target from.
+         * @param dir Side to follow target from.
+         * @sa follow_dir(), to_str(FollowDirection)
+         */
+        void set_follow_dir(FollowDirection dir);
+
+        /**
+         * @brief Sets dynamic filtering algorithm responsiveness to target movement.
+         * Lower numbers increase the responsiveness to changing long lat but also ignore less noise.
+         * @param responsiveness
+         * @return true (success) if responsiveness is in range (0.0 to 1.0), false (failure) otherwise.
+         * @sa responsiveness()
+         */
+        bool set_responsiveness(float resp);
+
+    private:
+        float _min_height_m = 8.0f; // Default & Minimum follow target height in meters.
```

Comment:
This is a public header files, it should not have anything private.

Reply:
Thanks. Changed